sky/catalog/common.py

```diff
@@ -476,8 +476,11 @@ def _filter_region_zone(df: 'pd.DataFrame', region: Optional[str],
 
 
 def get_instance_type_for_cpus_mem_impl(
```

Comment:
Can we add a unit test for this function with mocked data? i.e. avoid depending on specific cloud catalog to repro this edge case

Reply:
Sounds good, I've added unit tests.